knowledge-base/contextmenu-disable-focus.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Disable Focus on First Item in ContextMenu
+description: Learn how to prevent the first item in a context menu from automatically gaining focus by disabling or removing its default focus behavior.
+type: how-to
+page_title: Disable Focus on First Item in ContextMenu
+slug: contextmenu-kb-disable-focus
+tags: context-menu
+ticketid: 1622761, 1635324
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>ContextMenu for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+* How to remove autofocus from the first menu item?
+* How to prevent focus on the first item of the menu?
+* How to disable the focus on first context menu item?
+* How to stop ContextMenu recieves focus automaticallly when it shows?
+
+## Solution
+
+1. Set a custom CSS class to the ContextMenu through the `Class` parameter. This configuration will allow you to target specific ContextMenu instances.
+2. Use the defiend class to [Override the theme styles](https://docs.telerik.com/blazor-ui/styling-and-themes/override-theme-styles) by setting the CSS property **box-shadow** to **none**.
```

Comment:
Elements of the programming code and markup elements are rendered using the `monospace` font and not in **bold**. See the[ formatting guidelines](https://docs.telerik.com/style-guide/element-formatting#guidelines).
```suggestion
2. Use the defiend class to [Override the theme styles](https://docs.telerik.com/blazor-ui/styling-and-themes/override-theme-styles) by setting the CSS property `box-shadow` to `none`.
```